Health.py

```diff
@@ -1,39 +1,78 @@
-# For Healing:
-# vitC heals health
-# vitB heal magic
-# supplies repair mechanical
+FoodDict = {"Bread": [10, 0],
+            "Apple": [5, 0],
+            "Chicken Head": [20, 20],
+            "Thorny Fruit": [-20, -20]
+            }
 
-# food item would be ['foodName', 'food', [vitC_Value, vitB_Value]]
+RepairItems = {"Wrench": [10],
+               "Toolkit": [30],
+               "Repair Box": [100]
+            }
```

Comment:
What is the point of storing these numbers in lists? Do repair items only have supply points?

Reply:
I forgot that they didn't need to be put in lists, I'll remove them. And yes the only have supply points.